tests/contenttypes_tests/noop_migrations/0002_rename_foo.py

```diff
@@ -0,0 +1,12 @@
+from django.db import migrations
```

Comment:
this migration could be deleted and regenerated in initial migration.

Reply:
I split it in two migrations to handle different test cases. While having both changes in the same migration is doable I think it adds unnecessary complexity to the tests.

Reply:
ok understand